dbt_project_details.go

```diff
@@ -0,0 +1,91 @@
+package fivetran
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+// DbtProjectDetailsService implements the dbt project management, retrive dbt project details api
+// Ref. GET https://api.fivetran.com/v1/dbt/projects/{id}
+type DbtProjectDetailsService struct {
+	c            *Client
+	dbtProjectID *string
+}
+
+type DbtProjectDetailsdataBase struct {
+	ID            string `json:"id"`
+	GroupID       string `json:"group_id"`
+	CreatedAt     string `json:"created_at"`
+	CreatedById   string `json:"created_by_id"`
+	PublicKey     string `json:"public_key"`
+	DbtVersion    string `json:"dbt_version"`
+	GitRemoteUrl  string `json:"git_remote_url"`
+	GitBranch     string `json:"git_branch"`
+	DefaultSchema string `json:"default_schema"`
+	FolderPath    string `json:"folder_path"`
+	TargetName    string `json:"target_name"`
+}
```

Comment:
this could be replaced with common structure

Reply:
done. :)